New-Framework/Elements/AppLabel.cs

```diff
@@ -1,8 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿
 using TestStack.White.UIItems.Finders;
 
 namespace UnitTestProject1.Elements
```

Comment:
Modify it with a Text property
Because you need it into your form

Reply:
fixed